src/PointwiseFunctions/AnalyticSolutions/NewtonianEuler/CMakeLists.txt

```diff
@@ -4,8 +4,9 @@
 set(LIBRARY NewtonianEulerSolutions)
 
 set(LIBRARY_SOURCES
-    IsentropicVortex.cpp
-    )
+  IsentropicVortex.cpp
+  RiemannSolver.cpp
```

Comment:
I found the name rather surprising. I've also encountered these tests referred to as `Riemann Problem` in the literature and so would find that name a lot clearer. `RiemannSolver` to me sounds like something that should be a numerical flux.